safehttp/url.go

```diff
@@ -71,3 +73,17 @@ func (u URL) Port() string {
 func (u URL) Path() string {
 	return u.url.Path
 }
+
+// Parse parses a rawurl string into a URL structure.
+//
+// The rawurl may be relative (a path, without a host) or absolute
+// (starting with a scheme). Trying to parse a hostname and path
+// without a scheme is invalid but may not necessarily return an
+// error, due to parsing ambiguities.
```

Comment:
nit: The line breaking here seems a bit excessive. It's not even wrapped at 80 lines but at less.